tests/unit/test_beaker_transformer.py

```diff
@@ -1,5 +1,6 @@
 import pytest
 
+from mrack.context import global_context
```

Comment:
Shouldn't this be removed? I don't think it is needed. Or is it?

Reply:
you are correct thanks!